src/AvailableCommandsPacket.php

```diff
@@ -378,27 +389,61 @@ protected function putCommandData(CommandData $data, array $enumIndexes, array $
 		}
 	}
 
+	protected function initSoftEnumsInCommandData() : void{
```

Comment:
This should be directly below decodePayload(), since that's where it's actually relevant.